src/hotspot/share/opto/subnode.cpp

```diff
@@ -1899,3 +1899,50 @@ const Type* SqrtFNode::Value(PhaseGVN* phase) const {
   if( f < 0.0f ) return Type::FLOAT;
   return TypeF::make( (float)sqrt( (double)f ) );
 }
+
+static jlong reverse_bits(jlong val) {
```

Comment:
This function is wrong and/or misnamed.
It should be called reverse_bits_in_bytes.